bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/image/Icon.java

```diff
@@ -66,7 +83,20 @@ public IconType getType() {
      * @return current icon type
      */
     public IconType type() {
-        return iconBehavior.type();
+        return getType();
     }
 
+    @Override
+    public void onComponentTagBody(MarkupStream markupStream, ComponentTag openTag) {
+        super.onComponentTagBody(markupStream, openTag);
```

Comment:
I think ` super.onComponentTagBody(markupStream, openTag);` should be in the `else` clause

Reply:
You are right. Will change it.